Jenkinsfile-auto-deploy

```diff
@@ -0,0 +1,85 @@
+import groovy.json.JsonSlurperClassic
+
+def authHeader = "-H 'Authorization: token ${GITHUB_TOKEN}'"
+def contentTypeHeader = "-H 'Content-type: application/json'"
+def issueUrl = "https://api.github.com/repos/Wikia/design-system/issues/${params.pull_num}"
+def labels = ""
+def githubPagesBranch = "gh-pages"
+def publishPatchLabel = "publish-patch-on-merge"
+def publishMinorLabel = "publish-minor-on-merge"
+def publishMajorLabel = "publish-major-on-merge"
+def deployDocsLabel = "deploy-docs-on-merge"
+
+node('qa-executors') {
+	stage('Clone repo and validate') {
+		nodejs('v10 LTS') {
+			git credentialsId: 'bd3cf712-39a7-4b16-979e-ff86208ab2ea', url: 'git@github.com:Wikia/design-system.git', branch: "master"
+			currentBuild.description = "commit: ${params.commit}, PR: ${params.pull_num}"
+
+			stage('Check Pull Request labels') {
```

Comment:
If the job was triggered by the webhook `pull_request` event type it should receive the labels in the webhook data. See for example: [webhook](https://github.com/Wikia/pandora/blob/master/service/github-webhooks-proxy/src/main/resources/github-webhooks-proxy.yaml#L429) and [corresponding job](http://jenkins.wikia-prod:8080/job/feeds-and-posts-auto-deploy-marker/6843/parameters/). If it is the case for you here you may want to add the `labels` job parameterer and there'd be no need to curl it.

Although if it works and you are fine with it, we can keep it. ;)